.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+
+language: bash
+
+services:
+  - docker
+
+before_script:
+  - export APP_IMAGE=keboola-component
+  - docker -v
+  - docker build -t $APP_IMAGE .
+  - docker run -e AWS_S3_BUCKET -e AWS_REGION -e WRITER_AWS_ACCESS_KEY_ID -e WRITER_AWS_SECRET_ACCESS_KEY -e FIXTURES_AWS_ACCESS_KEY_ID -e FIXTURES_AWS_SECRET_ACCESS_KEY $APP_IMAGE composer ci
+
+  # push test image to ECR
```

Comment:
Buď není potřeba, nebo odcommentovat řádek 25 a dopsat integrační test

Reply:
jojo, intergrační test dopíšu, to je jasný

Reply:
jsou hotový